src/KINC.pri

```diff
@@ -33,11 +33,11 @@ isEmpty(LINK_LAPACKE) { LINK_LAPACKE = 1 }
 isEmpty(LINK_MPI_CXX) { LINK_MPI_CXX = 1 }
 
 # Include directories
-isEmpty($$(CUDADIR)) {
+isEmpty(CUDADIR) {
    CUDADIR = /usr/local/cuda
 }
 else {
-   CUDADIR = $$(CUDADIR)
+   CUDADIR = $$CUDADIR
```

Comment:
What is the purpose of this change? This code is intended to read the `CUDADIR` environment variable or else use a default, it looks like the new code would just always use the default.

Reply:
This was my mistake I did that wrong. Now it should work as I intended:

If the CUDADIR variable is set via qmake command line, it is left alone
Else it is set to the CUDADIR environment variable if one exists
Else it is set to a default